proto/cacheclient.proto

```diff
@@ -402,6 +402,8 @@ message _ListPopFrontRequest {
 message _ListPopFrontResponse {
   message _Found {
     bytes front = 1;
+    // length of the list after the pop
```

Comment:
@cprice404 - can you confirm that these responses will not break SDK's?

Reply:
I'm 90% sure that this is fine.  They will just not be deserialized by the SDKs until we update the SDK protos.

But, to be safe, we should just test by running integration tests in a dev cell and/or alpha when we have a working version of the corresponding ms2/mr2 changes.